### DISCUSSION TITLE
Set your New Year’s resolution with New to GitHub 🎉   [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#84145] [New to GitHub]
![Header of New to GitHub category check-in shows a purple starry background with shooting stars and clouds next to an octocat in a beret, striped shirt, and red pants holding a stack of books.](https://github.com/community/community/assets/92824518/3ff6f15b-be36-4fac-a2c8-e1bc3ba92918)

A new year is a perfect opportunity to set goals, intentions, and resolutions and if your goal is to uplevel your GitHub expertise, the New to GitHub category is the place to be! 

### 🌟Community Superstars: A round of applause for these community citizens 👏 
* @shiskhare: For [their great suggestions](https://github.com/orgs/community/discussions/82718) on how @gv29847 could share their work and by recommending an easy to follow video. Also shoutout to @gv29847 for joining the world of open source and sharing their work 😃. 
* @faridnec: Loved their approachable answer and follow-up on [this post](https://github.com/orgs/community/discussions/82699) about setting up repositories for a multi-platform project. Thanks for sharing your perspective! 
* @ananuness: Has been helpful around the category lately and we’re so appreciative ❣️. They did some great troubleshooting [in this discussion](https://github.com/orgs/community/discussions/81471) - be sure to check it out. 

### ⬆️ Level up: Resources to take your skills to new heights for the new year
* **Global Hack Week - Beginner’s Week**: As part of [Major League Hackers’s](https://ghw.mlh.io/) monthly Global Hack Week, January 5th - 11th, 2024 will be [Beginners Week](https://ghw.mlh.io/events/beginners-week2)! Join them for a week of beginner-friendly content to kick off 2024! 🎇
Global Hack Week is a 100% free event for anyone and it's easy to register and participate - follow along with [live challenges](https://ghw.mlh.io/challenges). 👍
* **GitHub Skills**: [Learn how to use GitHub with interactive courses](https://skills.github.com/) designed for beginners and experts. I spent this week brushing up on my Pages skills. 
* **GitHub Learning Pathways**: Whether you’re a developer, engineering leader or GitHub administrator, there's a path for you to explore. GitHub’s Learning Pathways are self-paced tutorials designed to help you grow from beginner to expert. [Find your path](https://resources.github.com/learn/pathways/). 

### 📆 2023 Wrap Up: Moments, members, and more from 2023 we don’t want you to miss!
* **December 2023 New to GitHub Check-in**: We celebrated our top contributing members to the category and solves in the [final check-in on the year](https://github.com/orgs/community/discussions/79132). 
* **GitHub Community** - Here's to an incredible 2023: Check out all we achieved together and celebrate the past year with [2023 Community Discussions summary](https://github.com/orgs/community/discussions/84131). 
* **GitHub’s top blog posts of 2023**: With the books closed on 2023, we're highlighting some of the incredible work from GitHub’s engineers, product teams, and security researchers in [this article](https://github.blog/2023-12-27-githubs-top-blog-posts-of-2023/).

Wishing everyone a magical 2024 full of learning, discovering new projects, and joy! What are you hoping to learn this year?

**Comment by ananuness** [comment]
How sweet!! 💖💖💖 Thank you very much for your care and recognition, I hope this year is wonderful for all of us, full of achievements and learning! 🌱

And for this year, I want to learn e2e testing, more React hooks, advanced Nest and a little more about AWS services... it's going to be quite a year hehe 😶‍🌫️

**Comment by wesleybertipaglia** [comment]
Congratulations for all in the community 🎉🎉🎉. 

The past year I have learned a lot by practice and enjoying in code communities, in this year I go to make the same, but more, and I want to deep in design patterns, architecture and learn more about scalability 🚀

**Comment by queenofcorgis** [comment]
Keep the new year energy going in the newest [New to GitHub check-in](https://github.com/orgs/community/discussions/88727)!
